docs/PublishingGuide.md

```diff
@@ -0,0 +1,86 @@
+Try accessing your site again when a green tick appears beside your fork.
+
+### Keeping your site up-to-date with your code contribution
+
+[Travis-CI](https://travis-ci.org/) offers `Cron Jobs` in intervals of daily, weekly or monthly.
+
+1. Login to [Travis-CI](https://travis-ci.org/)
+1. Go to [your fork in Travis-CI](https://travis-ci.org/search/publish-RepoSense/), click on **More options** on the right then access **Settings**
+1. Under **Cron Jobs**, choose **master** for `Branch`, **Always run** for `Options` and pick an `Interval` of your choice; <br/>
+then click **Add**
+![Travis-CI Cron](images/publishingguide-cronsetting.jpg "Travis-CI Cron")
+
+Alternatively, you can manually trigger an update.
+
+1. Go to [your fork in Travis-CI](https://travis-ci.org/search/publish-RepoSense/), click on **More options** on the right then **Trigger build**
+1. In the pop up, click **Trigger custom build**
+
+### Specifying which version of RepoSense to use
+
+As RepoSense is being actively developed, its master branch is frequently updated with new features and fixes. <br/>
+For stablility or familiarity, you may want to use the release. <br/>
+
+#### Use our latest release (Stable)
+
+You can find the changelog of latest release [here](https://github.com/reposense/RepoSense/releases/latest).
+
+Edit line 10 of [run.sh](../../../../publish-RepoSense/edit/master/run.sh) to use `--release`. <br />
+```
+10   ./get-reposense.py --release
+```
+
+#### Use our master branch (Beta)
+
+You can find the list of commits to master branch [here](https://github.com/reposense/RepoSense/commits/master) and planned features/fixes [here](https://github.com/reposense/RepoSense/milestones).
+
+Edit line 10 of [run.sh](../../../../publish-RepoSense/edit/master/run.sh) to use `--master`. <br />
+```
+10   ./get-reposense.py --master
+```
+#### Use a specific version of release
+
+Be sure to read and understand [all breaking changes and bugs](https://github.com/reposense/RepoSense/releases) before proceeding.
+
+Edit line 10 of [run.sh](../../../../publish-RepoSense/edit/master/run.sh) to use `--tag TAG`, where `TAG` is the [target version of release](https://github.com/reposense/RepoSense/tags). <br />
+```
+10   ./get-reposense.py --tag v1.6.1
+```
+### Managing your configuration files with the help of collaborators
+
+If you use RepoSense to monitor a large number of collaborators, it may be more systematic to have them update their git configurations into your fork using pull-requests.
+
+To ensure that their pull-requests are correct, we can use use Netlify to preview how the changes would look like in production, before deploying them into GitHub Pages.
+
+1. Sign up and login to [Netlify](https://www.netlify.com/)
+1. Use this [link](https://app.netlify.com/start) to create a new site
+1. Select **GitHub** as your Git provider and authorize Netlify in the pop up if prompted
+1. Choose `publish-RepoSense` to deploy <br/>
```

Comment:
We may need to install Netlify on GitHub 1st before this step, and to choose to install on publish-RepoSense repository

Reply:
Rephrase this step to better capture two different scenarios.

However, Netlify's setup wizard is buggy. It got stuck at a white page after install Netlify on the fork for the first time. Need to refresh the main page to continue. 🤔